### PR TITLE
Quadratic Equation

### PR DESCRIPTION
**Pull request by prasadvpatil**
issue #91 
Program to solve the quadratic equation

### CODE REVIEW

File: Basic-Scripts/quad_equation.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Solve the quadratic equation ax**2 + bx + c = 0
+# import complex math module
+import cmath
```
Comment (SKAUL05) [inline Basic-Scripts/quad_equation.py]: Include `main` in your script.
Follow PEP8 Guidelines.
Use docstrings

**Review by SKAUL05 — CHANGES_REQUESTED** [review]

File: Basic-Scripts/quad_equation.py (hunk 1 above)
Comment (prasadvpatil) [reply Basic-Scripts/quad_equation.py]: @SKAUL05 i have made changes as per your guidance, please review it.
thank you.

File: Basic-Scripts/quad_equation.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        print("Two Distinct Complex Roots Exists: ")
+        print("r1=%.2f+%.2f and r2=%.2f-%.2f" % (root_1, imaginary, root_2, imaginary))
+
+# Main Program
+if __name__ == "__main__":
+  a = float(input("Please Enter a Value of a Quadratic Equation : "))
+  b = float(input("Please Enter b Value of a Quadratic Equation : "))
+  c = float(input("Please Enter c Value of a Quadratic Equation : "))
+
+print(findRoots(a, b, c))
```
Comment (SKAUL05) [inline Basic-Scripts/quad_equation.py]:
```suggestion
    print(findRoots(a, b, c))

```

**Review by SKAUL05 — CHANGES_REQUESTED** [review]

File: Basic-Scripts/quad_equation.py (hunk 1 above)
Comment (prasadvpatil) [reply Basic-Scripts/quad_equation.py]: I did the changes now u suggested too. @SKAUL05

File: Basic-Scripts/quad_equation.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        print("Two Distinct Complex Roots Exists: ")
+        print("r1=%.2f+%.2f and r2=%.2f-%.2f" % (root_1, imaginary, root_2, imaginary))
+
+# Main Program
+if __name__ == "__main__":
+  a = float(input("Please Enter a Value of a Quadratic Equation : "))
+  b = float(input("Please Enter b Value of a Quadratic Equation : "))
+  c = float(input("Please Enter c Value of a Quadratic Equation : "))
+
+    print(findRoots(a, b, c))
```
Comment (SKAUL05) [inline Basic-Scripts/quad_equation.py]: Wrong indentation

Comment (SKAUL05) [reply Basic-Scripts/quad_equation.py]: Check once properly

**Review by SKAUL05 — CHANGES_REQUESTED** [review]

**Review by SKAUL05 — APPROVED** [review]
Good to go :)

**Pull request merged**